src/catalog.ts

```diff
@@ -48,6 +51,12 @@ export class Catalog {
           new Dataset(
             new IRI(bindings.get('?dataset').value),
             bindings.get('?name').value,
+            [
+              new Organization(
```

Comment:
Also up for debate: are organizations the only creators or can individual persons be creating agents as well? For now, we only have organizations.

Reply:
Agree!